Week1/Solution Week-1/Solution-7.py

```diff
@@ -0,0 +1,17 @@
+#7.Write a python script to enter any string, replace vowel with its position number.
+#For Example: input: Amit
+#             output:0m2t
+
+def vowelchange(s):
+    b=''
+    a=('a','e','i','o','u','A','E','I','O','U')
+    for index,j in enumerate(s):

```

Comment:
Can you write other method for this code in which you can not use enumerate( ) function?

Reply:
yes. we use statements like below.
for i in range(len(name)):
    if name[i] in ['a','e','i','o','u']:
        new_name+=str(i)
    else:
        new_name+=name[i]
thank you.